docs/asciidoc/configuration.asciidoc

```diff
@@ -273,7 +273,7 @@ port: 9200
 -----------
 
 The default is `9200`.  This value will only be applied to <<hosts,hosts>>
-without a port affixed, e.g. `localhost:9202`.
+without a port affixed, e.g. `localhost`.
```

Comment:
If you really want to change this, I request that you change it instead to show a before and after, e.g.

"This value will only be applied to hosts _without_ a port affixed. For example, if you have:

```
  hosts: "localhost:9200"
  port: 9202
```
In this example, port `9202` will not be affixed to `localhost:9200`, nor will `9202` replace the `9200`."